packages/apps/human-app/frontend/src/modules/auth/providers/require-auth.tsx

```diff
@@ -3,7 +3,7 @@ import { createContext } from 'react';
 import { useAuth } from '@/modules/auth/hooks/use-auth';
 import { routerPaths } from '@/router/router-paths';
 import type { AuthenticatedUserContextType } from '@/modules/auth/context/auth-context';
-import { PageCardLoader } from '@/shared/components/ui/page-card-loader';
+import { PageCardLoader } from '@/shared/components/ui/page-card';
```

Comment:
why do we have a PageCardLoader component in the `page-card` file? I think the name of the component should be exactly like the name of the file